Source/SwiftLintFramework/Rules/Style/ExplicitReturnRule.swift

```diff
@@ -0,0 +1,142 @@
+import Foundation
+import SourceKittenFramework
+import SwiftSyntax
+
+public struct ExplicitReturnRule: ConfigurationProviderRule, SubstitutionCorrectableRule, OptInRule {
+    public var configuration = ExplicitReturnConfiguration()
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "explicit_return",
+        name: "Explicit Return",
+        description: "Prefer explicit returns in closures, functions and getters.",
```

Comment:
do you want to handle subscripts here too?

Reply:
I'd like to, but I wonder if I should to this in this PR. The reason is, the opposite, upstreamed rule: `implicit_return` only works for closures, functions and getters as well. Preferably, we should add subscripts support there as well, and in my opinion - I'd rather do this separately. Let me know what do you think. If you think it should be done in this PR, I can handle it!